.github/workflows/web-prod.yml

```
@@ -0,0 +1,58 @@
+on:
+  release:
+    types: [released]
+
+jobs:
+  release:
+    runs-on: blacksmith-4vcpu-ubuntu-2404
+    permissions:
+      contents: 'read'
+      id-token: 'write'    
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)